### DISCUSSION TITLE
Favorites tab above the tabs

### DISCUSSION THREAD

**Discussion by luiggircardoso** [zen-browser/desktop#2355] [Ideas]
I think it would be a pretty cool feature. The favorites tab would be basically the pinned tabs, but when you close that tab, it would go back to it's original url. And also make it so it's not open forever because of waste of ram and CPU in our PCs.

![image](https://github.com/user-attachments/assets/33235879-ecb9-43d2-8e15-cb1902457a42)

**Comment by keifererikson** [comment]
Really hoping this feature can be implemented. I'm so used to this in Arc Browser. I'm trying to make the full switch to Zen but this functionality is one of the last things holding me back from jumping ship. I made a discussion about this a while ago #916

**Reply by luiggircardoso** [reply]
Same! I just switched and i am missing this and the folders.
